### DISCUSSION TITLE
Como ser mais produtivo com tanto conteúdo pra estudar e acompanhar?

### DISCUSSION THREAD

**Discussion by cadudias** [frontendbr/forum#429] [Discussão]
Olá, pessoal!

Uma das coisas que eu sinto dificuldade na nossa área é dividir o tempo pra estudar tanta coisa, hoje já melhorei bastante nisso e o que me ajudou foi usar aceleradores de texto, vídeo e áudio.

Acelerar pra 1.5X ou até 2X me ajudou a acompanhar canais do YT e podcasts da área.

Mas quero saber o ponto de vista de vocês, como fazem pra lidar com a montanha de coisas pra estudar além de, muitas vezes, trabalhar full time.

Vou deixar uns links dos aceleradores que eu uso, caso tenham melhores me mandem :)

- [Youtube Playback Speed Control](https://chrome.google.com/webstore/detail/youtube-playback-speed-co/hdannnflhlmdablckfkjpleikpphncik)

- Podcasts - [BeyondPod](https://www.beyondpod.mobi/android/index.htm) (na versão PRO dá pra acelerar, por 16 reais valeu a pena pra mim)

- Textos - [Reedy](https://play.google.com/store/apps/details?id=azagroup.reedy&amp;hl=en) (já consegui ler quase 70 artigos por mês colocando em 250/300 palavras por minuto) 

Valeu!

**Comment by joao-alberto** [comment]
Vídeos eu sempre vejo acelerados também, mas com a configuração do próprio YouTube.

Mas para leitura nunca tinha usado nada, vou testar esse Reedy pra ver como é, valeu a dica!

**Comment by Cleysonlb** [comment]
Eu uso muito o https://getpocket.com/ pra salvar artigos e ler offline, atualmente não to conseguindo acompanhar muito conteudo mas é por falta de organização haha

**Comment by akfzambrana** [comment]
Como é que vc entende passando em maior velocidade? (eu nunca tentei, fiquei curiosa)
> Acelerar pra 1.5X ou até 2X me ajudou a acompanhar canais do YT e podcasts da área.

**Comment by cadudias** [comment]
A gente se acostuma, @akfzambrana . Inclusive isso me ajuda a me concentrar mais no que as pessoas estão falando e principalmente na leitura de textos. Existem estudos que dizem que ao contrário do que se pensa ler mais rápido aumenta a concentração. Começa com 1.25X, depois de algumas semanas vai ver que assistir vídeos no 1.0 é uma tortura aahuuha. Dai vai aumentando aos poucos, algumas pessoas falam bem rápido dai prefiro escutar com 1 ou 1.25, se a pessoa fala devagar ou está escrevendo código, o que costuma ser mais lento, dá pra aumentar pra 1.5 ou 2 tranquilamente.

**Comment by cadudias** [comment]
@Cleysonlb parece interessante, valeu pela dica!

**Comment by onhernandes** [comment]
@akfzambrana @kadoohd eu comecei ouvindo podcasts em 1.25X, mas achei que podia aumentar e joguei pros 2X de uma vez, no mesmo dia, eu gostei e não foi difícil de entender(lógico, isso também depende de quem tá falando e como ta falando). Dependendo do podcaster eu deixo mais lento(raramente acontece, mas isso é pra conseguir entender). Hoje em dia eu ouço entre 2.2X ~ 2.5X. Aumentar o volume e usar fones auriculares ajuda também, já que remove boa parte dos ruídos externos.

**Comment by cadudias** [comment]
@mhernandes valeu! To tentando aumentar as poucos mas pra ouvir em inglês complica, por enquanto só consigo no 1.5. Aos poucos chegamos lá.

**Comment by felipefialho** [comment]
@kadoohd Pior que faz sentido, leio muito rápido, quase uma leitura dinâmica das coisas apesar de manter bom nivel de atenção.

**Comment by rberts** [comment]
Para vídeo eu uso o [Speed Controller](https://github.com/igrigorik/videospeed) e para artigos o Pocket também.

@kadoohd Esse que você usa eu não sei se vale só pra youtube (pelo nome), mas o speed controller vale pra qualquer tipo de vídeo que tenha na página.

Uso muito o [Feedly](https://feedly.com/) também pra acompanhar o conteúdo de vários sites, e salvando no bookmark (ou no pocket) para ir lendo depois.

**Comment by cadudias** [comment]
@istiu Tenho usado o feedly também, principalmente pra companhar os blog do pessoal que eu sigo. O Pocket realmente é uma boa, várias vezes fiquei sem 3g e queria ler alguma coisa, já aderi :)

**Comment by cadudias** [comment]
@LFeh um grande problema de ler devagar ou sem programas como o reedy é que a nossa visão fica pulando pra frente e pra trás ao longo do texto, às vezes a gente lê 10 vezes o mesmo trecho por culpa da falta de atenção. Com a leitura rápida isso não acontece, muita gente passa o dedo no texto aumentando gradativamente a velocidade pra que a visão acompanhe e não se perca. Claro, conteúdos técnicos e complexos eu leio várias vezes e com calma pra entender.

**Comment by onhernandes** [comment]
@kadoohd sobre essa questão de ler rápido, eu costumo aumentar a fonte do artigo pelo Pocket, isso me força à rolar a página com mais frequência(praticamente toda hora) e eu consigo ler mais rápido sem forçar a visão
